src/AWS.Deploy.Orchestration/EnvironmentVariableKeys.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+namespace AWS.Deploy.Orchestration
+{
+    /// <summary>
+    /// Contains shared environment variable keys
+    /// </summary>
+    public static class EnvironmentVariableKeys
```

Comment:
Would this make more sense in Common?  Is it specific to Orchestrator?

Reply:
it is also used in CLI but given it has reference to Orchestrator it works. It could be moved to Common. I will do it.